api/src/backend/tasks/jobs/scan.py

```diff
@@ -526,7 +527,7 @@ def create_compliance_requirements(tenant_id: str, scan_id: str):
         with rls_transaction(tenant_id):
             scan_instance = Scan.objects.get(pk=scan_id)
             provider_instance = scan_instance.provider
-            prowler_provider = initialize_prowler_provider(provider_instance)
+            prowler_provider = return_prowler_provider(provider_instance)
```

Comment:
I'm curious about this, is it returning an instance of the SDK's provider class? What if the task is run in a different container?

This could be totally wrong, just in case.